File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int len=nums.length,max=0,temp=0;
        for(int i=0;i<len;i++){
            if (nums[i]==1){
                temp+=1;
            }
            else{
                if(temp>max){
                    max=temp;
                }
                temp=0;
            }
        }
        if(temp>max){
            max=temp;}
        return max;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        if len(nums)==1 and nums[0]==1:
            return 1
        if len(nums)==1 and nums[0]==0:
            return 0
        count=0
        temp=0
        for i in nums:
            if i==1:
                temp+=1
                count=max(temp,count)
            else:
                temp=0
        return count